src/charm.py

```diff
@@ -95,6 +143,107 @@ def _pebble_layer(self) -> ops.pebble.LayerDict:
             },
         }
 
+    @property
+    def _application_bind_address(self) -> str | None:
+        binding = self.model.get_binding("juju-info")
+        if not binding:
+            return None
+        if not binding.network:
+            return None
+        if not binding.network.bind_address:
+            return None
+        return str(binding.network.bind_address)
+
+    ## Status Checks ##
+    def _storages_attached(self) -> bool:
+        """Return if the storages are attached."""
+        return self.model.storages.get("config") and self.model.storages.get("database")  # type: ignore
```

Comment:
Why the type ignore here?

Reply:
For some reason, static checks don't believe that self.model.storages.get() returns a boolean value, even though if there's no storage it will return None.

I tried some different things, and casting them to bool seems to work so its removed.